elements/stagger.lua

```diff
@@ -94,19 +74,46 @@ local function Update(self, event, unit)
 			end
 		end
 	end
+end
+
+local function Update(self, event, unit)
+	if(unit and unit ~= self.unit) then return end
+
+	local element = self.Stagger
+
+	--[[ Callback: Stagger:PreUpdate()
+	Called before the element has been updated.
+
+	* self - the Stagger element
+	--]]
+	if(element.PreUpdate) then
+		element:PreUpdate()
+	end
+
+	local cur = UnitStagger('player')
+	local max = UnitHealthMax('player')
+	local perc = cur / max
+
+	element:SetMinMaxValues(0, max)
+	element:SetValue(cur)
+
+	--[[ Override: Stagger:UpdateColor(unit, perc)
+	Used to completely override the internal function for updating the widgets' colors.
+
+	* self - the Stagger element
+	* perc - the percentage of staggered damage (number)
```

Comment:
It can be relative to what matters to whoever overrides UpdateColor. Also not all ppl think of percentages as values between 0-1.